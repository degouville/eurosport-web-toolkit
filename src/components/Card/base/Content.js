import React from 'react';
import styled from 'react-emotion';
import PropTypes from 'prop-types';

const StyledContent = styled('div')`
  background-color: #10121e;
  padding: 6px 10px;
  line-height: 1.3em;
`;

const Content = ({ children }) => <StyledContent>{children}</StyledContent>;

Content.displayName = 'Content';

Content.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Content;
