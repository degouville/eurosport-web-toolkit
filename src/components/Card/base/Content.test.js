import React from 'react';
import { shallow } from 'enzyme';
import Content from './Content';

it('renders a CardContent', () => {
  expect(shallow(<Content>CardContent</Content>)).toMatchSnapshot();
});
