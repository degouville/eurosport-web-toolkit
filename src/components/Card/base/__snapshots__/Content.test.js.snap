// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a CardContent 1`] = `
<Styled(div)>
  CardContent
</Styled(div)>
`;
